### DISCUSSION TITLE
google tv problem

### DISCUSSION THREAD

**Discussion by Masoud-1990** [strongswan/strongswan#1977] [Ideas]
app is working excelent in my android phone and I can reach an amazing speed (600mbps+) which I couldn't with other VPN protocols.
I want to use it in my Sony google TV for streaming but the problem is there is no exclusive strongswan android/google TV app. I managed to sideload and install it in my TV but the app icon is not showing in the list and also when I open it from the app manager in settings it's speed is awfully limited and is unstable.
I believe that creating an android TV app is not a big deal for the developer?
I currently use shadowsocks TV app but I couldn't get higher than 300mbps with this protocol on my server.

**Comment by tobiasbrunner** [comment]
Basically a duplicate of #210.  Also, I don't think there is anything we could do differently about the speed or stability.
